serializations.md

```diff
@@ -732,21 +753,58 @@ tree.
 
 ```
 
+###### Channel off-chain pinned block for on-chain environment
+
+Each update can be optionally based on an on-chain block that is used for an
+environment for on-chain lookups: accounts, oracles queries, responses and
+etc. This is to improve the process of both reaching an off-chain conesus
+regarding changes and having more predictable on-chain forced progress
+executions.
+
+```
+[ <hash>      :: binary()
```

Comment:
I see that only ambiguity of this is whether this is key or micro block. And that ambiguity is resolved with bespoke type field. This could have been solved with `id()` type or similar, though that could have been overengineered. Ok.

Reply:
Well yes, but `id()` are just IDs and it felt wrong.